.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  presets:["es2015", "stage-0", "react"]
+  presets:["es2015", "stage-0", "react"],
```

Comment:
Trailing commas makes version-control diffs cleaner, you can simply add a new line without modifying the previously last line ...